### PR TITLE
Create ~/.ssh if necessary

### PR DESCRIPTION
**Pull request by native-api**
https://travis-ci.community/t/pushing-to-github-from-windows-host-key-verification-failed/9495

### CODE REVIEW

File: lib/travis/build/addons/ssh_known_hosts.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def config
           def add_ssh_known_hosts
             sh.fold 'ssh_known_hosts.0' do
               sh.echo "Adding ssh known hosts", ansi: :yellow
+              sh.cmd 'mkdir -p "${TRAVIS_HOME}/.ssh"'
```
Comment (BanzaiMan) [inline lib/travis/build/addons/ssh_known_hosts.rb:28]: We should be more uniform in use of certain methods, but here, we can use:

```ruby
sh.mkdir "${TRAVIS_HOME}/.ssh", recursive: true
```

Comment (native-api) [reply lib/travis/build/addons/ssh_known_hosts.rb]: Sure. Didn't think of it since some code below used a raw command for mkdir.

Comment (BanzaiMan) [reply lib/travis/build/addons/ssh_known_hosts.rb]: Right. We should do better with respect to the uniformity throughout the code base.

**Comment by annamajorova0278** [comment]
@ #1964

**Comment by annamajorova0278** [comment]
296b15

**Comment by annamajorova0278** [comment]
### 1964

**Pull request merged**